utils/setup.php

```diff
@@ -70,10 +72,6 @@
     $iInstances = 1;
     warn("resetting threads to $iInstances");
 }
-if ($iInstances > getProcessorCount()) {
-    $iInstances = getProcessorCount();
-    warn("resetting threads to $iInstances");
-}
```

Comment:
Unrelated.

Reply:
This is actually directly related. In the case where `setup.php` is running on a host other than the database server, it is incorrect for the processor count on the local host to be used to limit the number of concurrent queries allowed to run on the database server. Engineering a way for the PHP script to ask the database server for a processor count seemed excessive, not to mention a way to infer the local/remote status of the database server from the result of `DB::parseDSN(CONST_Database_DSN)`, so I chose to allow `setup.php` to trust the value provided on the command line.

Reply:
Makes sense.